cookbook/core/control_flow/chain_tasks.py

```diff
@@ -66,8 +60,34 @@ def chain_tasks_wf() -> pd.DataFrame:
 #
 #       create_node(task_name, parameter1=argument1, parameter2=argument2, ...)
```

Comment:
Since we don't use `create_node` anymore, let's remove this note.

Reply:
Done.